process/project-lifecycle.md

```diff
@@ -159,6 +159,16 @@ When voting on a proposal to archive a project, TAC members may consider:
 * If the project's inactivity or inconsistent maintenance presents a user security risk.
 * If the project's design or approach is no longer a recommended best practice.
 
+#### Considerations when Archiving a TI that has sub-efforts embedded within ####
+
+1.) Overseeing TI will be moving to "Archived" status
+2.) Active subgroups (SIG or project) is still active and desires to continue collaborating
+3.) Subgroup reviews active TIs to see if there is alignment of vision and purpose (https://openssf.org/community/openssf-working-groups/)
+4.) Subgroup meets with TI leader and group to discuss adoption
+5.) New TI agrees to become new oversight group for subgroup
+6.) PRs filed in TI and TAC repos to note change of status
+7.) Archiving TI has PR filed noting new status
```

Comment:
I like this list, but I might split this into steps/tasks for the hosting TI vs. the sub-TI that wants to transition. I also think that'll also make it clearer about who's responsible for which of these tasks (TAC vs. TI chairs, vs sub-TIs).